web_handler_block.py

```diff
@@ -26,6 +46,11 @@ class WebHandler(GeneratorBlock):
                              default='',
                              allow_none=True)
 
+    cors = ObjectProperty(CORS,
+                          title="Access-Control Headers",
+                          default=dict(),
```

Comment:
I think this needs to be `default=CORS()` for object property defaults.

Reply:
i was going to ask about this, wasn't sure what it was supposed to be.